projects/step-core/src/lib/modules/basics/shared/generate-api-key-strategy.ts

```diff
@@ -1,3 +1,7 @@
+import { Observable } from 'rxjs';
+
 export interface GenerateApiKeyStrategy {
-  showGenerateApiKeyDialog(): void;
+  showGenerateApiKeyDialog(): Observable<any>;
+  getServiceAccountTokens(): Observable<Array<any>>;
+  revoke(id: string): Observable<any>;
```

Comment:
```suggestion
  revoke(id: string): Observable<void>;
```
As I see, we're not interested in the result value of this method and don't analyze it. Let's make it `void`